projects/pgai/pgai/vectorizer/vectorizer.py

```diff
@@ -137,7 +137,7 @@ class Vectorizer(BaseModel):
     source_pk: list[PkAtt]
     queue_failed_table: str | None = None
     errors_schema: str = "ai"
-    errors_table: str = "vectorizer_errors"
+    errors_table: str = "_vectorizer_errors"
```

Comment:
I'm not 100% convinced about this change. AFAIK `errors_table` is not configurable when creating a vectorizer yet. If we end up allowing this, we would need to create the error views for each configured errors_table.

Reply:
this change is correct

Reply:
Yeah, technically it is at this moment. Just wanted to leave a statement here for reference.